### PR TITLE
Book Show Page Stats

### LINKED ISSUE #24: User Story 11, Book Show Page has Statistics
```
As a Visitor,
When I visit a book's show page,
I see an area on the page for statistics about reviews:
- the top three reviews for this book (title, rating and user only)
- the bottom three reviews for this book  (title, rating and user only)
- the overall average rating of all reviews for this book
```

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [ ] edge case ("sad path") tests written
- [ ] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #21

child of #33

### PR DESCRIPTION
**Pull request by alexander-mathieu**
This PR adds a section to the book show page that displays the top three worst/best reviews for a book, as well as the average rating.

- Add additional test data to book model spec
- Add specs for lowest/highest three reviews to book model spec
- Add lowest/highest_three_reviews instance method to book reviews
- Implement lowest/highest three reviews in book show view
- Implement average rating in book show view

Resolves #24

### CODE REVIEW

File: app/views/books/show.html.erb [hunk 1]
```diff
@@ -8,8 +8,32 @@
...
+      <div id="low-review-<%= review.id %>">
+        <h4><%= review.title %>, <%= review.rating %> Stars</h4>
+        <%= link_to review.user.name, user_path(review.user.id) %>
+      </div>
+      <% end %>
+  </section>
+</section>
+<br>
+
+<%= link_to "Delete Book", book_path(@book), method: :delete, data: {confirm: "Are you sure you want to delete the book?"} %>
```
Comment (BrennanAyers) [inline app/views/books/show.html.erb:33]: Almost lost ya!

File: spec/models/book_spec.rb [hunk 2]
```diff
@@ -16,6 +16,10 @@
     before(:each) do
       @user_1 = User.create!(name: "Anony-moose")
       @user_2 = User.create!(name: "VinnyCheese")
+      @user_3 = User.create!(name: "LogyBear")
+      @user_4 = User.create!(name: "MILLS")
+      @user_5 = User.create!(name: "DocPat")
+      @user_6 = User.create!(name: "BilhamTheConqueror")
```
Comment (BrennanAyers) [inline spec/models/book_spec.rb:22]: 👍
